packages/server/src/nfts/nfts.controller.ts

```diff
@@ -29,9 +29,11 @@ export class NftsController {
     @Body() body: MintNftRequest,
   ) {
     const payload: CoreapiMintNFTRequest = {
-      documentId: body.document_id,
-      proofFields: body.proof_fields,
-      depositAddress: body.deposit_address,
+      // @ts-ignore
```

Comment:
Is this a linter ignore check? If so, what is the "correct" way of defining variables in TS?

Reply:
so this is unfortunately related to #442, I need to look into another way/config to generate the API so that it is in the right casing